main.go

```diff
@@ -116,6 +116,7 @@ func createKubeClient(apiserver string, kubeconfig string) (clientset.Interface,
 		return nil, err
 	}
 
+	config.UserAgent = fmt.Sprintf("%s", version.GetVersion())
```

Comment:
`fmt.Sprintf` is not necessary here, `version.GetVersion()` already returns a string

Reply:
Good catch. Will do.